include/fmt/chrono.h

```diff
@@ -664,6 +664,30 @@ enum class numeric_system {
   alternative
 };
 
+// Glibc extensions for formatting numeric values.
+enum class pad_type {
+  unspecified,
+  // Do not pad a numeric result string.
+  none,
+  // Pad a numeric result string with zeros even if the conversion specifier
+  // character uses space-padding by default.
```

Comment:
Which conversion specifiers use space padding by default?

Reply:
For example, the command `date "+%10a"` prints `       Tue` on Ubuntu 22.04.1. 
If we were to implement the glibc extension for the optional width, we might need to consider this padding. Otherwise, we don't have space padding in the current codebase.